.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: UI Tests
+
+on:
+  workflow_dispatch:
+  deployment_status:
+
+env:
+  GIT_PAT: ${{ secrets.GIT_PAT }}
```

Comment:
is the access token needed output of the git clone step? its probably best not to have it around the environment if its not needed.